plugins/push/frontend/public/templates/push-notification-tab.html

```diff
@@ -75,15 +64,21 @@
                         </el-select>
                     </template>
                     <template v-slot="scope">
-                        <el-table-column type="clickable" sortable="custom" prop="name" :label="i18n('push-notification.table-message-name')" class="cly-vue-push-notification-multi-line-table-data">
+                        <el-table-column fixed width="400" type="clickable" sortable="custom" prop="name" :label="i18n('push-notification.table-notification-name')" class="cly-vue-push-notification-table-column">
                             <template slot-scope="scope">
-                                <span class="cly-vue-push-notification-multi-line-table-data__first-line" @click="onRowClick(scope.row)">{{scope.row.name}}</span>
-                                <br />
-                                <span class="cly-vue-push-notification-multi-line-table-data__second-line">{{scope.row.platforms.map(function(platform){return platform.label}).join(", ")}}</span>
+                                <div class="bu-is-flex bu-is-flex-direction-column">
+                                    <span class="cly-vue-push-notification-table-column__first-line" @click="onRowClick(scope.row)">{{scope.row.name}}</span>
+                                    <span class="cly-vue-push-notification-table-column__second-line">
+                                        <span> {{scope.row.platforms.map(function(platform){return platform.label}).sort(function(a,b){ return a < b}).join(", ")}} </span>
```

Comment:
I'm not against at naive one liners, but this is a bit complicated :laughing:

Reply:
Yeah, functional programming has become a weak point over the last couple of years so :expressionless: Unfortunately <es6 is not doing me favours here. I will add a method.